central/main.go

```diff
@@ -284,11 +285,33 @@ func main() {
 	pkgMetrics.NewDefaultHTTPServer(pkgMetrics.CentralSubsystem).RunForever()
 	pkgMetrics.GatherThrottleMetricsForever(pkgMetrics.CentralSubsystem.String())
 
+	if env.PrivateDiagnosticsEnabled.BooleanSetting() {
+		privateServer := private.NewHTTPServer()
+		privateServer.AddRoutes(privateRoutes())
+		privateServer.RunForever()
+	}
+
 	go startGRPCServer()
 
 	waitForTerminationSignal()
 }
 
+func privateRoutes() []*private.Route {
+	result := make([]*private.Route, 0)
+	for r, h := range routes.DebugRoutes {
+		result = append(result, &private.Route{
+			Route:         r,
+			ServerHandler: h,
+			Compression:   true,
+		})
+	}
+	result = append(result, &private.Route{
```

Comment:
The question might be obsolete, but did we make any decision on the authZ part of this yet? Right now its without any authorization AFAICT and it's definitely done on purpose, so I assume we are sticking to the "controlling access via port forward" solution?

Reply:
Correct, we control access via access to the dataplane with the level of permissions that allows doing user one of 3 things:
* `kubectl port-forward` to central pod
* `kubectl exec` into central pod
* modifying central service definition

Reply:
Sounds good to me for the beginning, let's also add a comment there so other folks know the context as well and aren't surprised why there's no authorizer.